docs/src/examples/components/Button/SplitButton/SplitButtonExample.shorthand.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react'
+import { SplitButton } from '@stardust-ui/react'
+
+const handleMainOptionClick = e => {
+  alert('PDF is all around us!')
```

Comment:
this is not triggered when using keyboard and pressing `Enter\Space` on the whole split button

Reply:
addressed